sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -945,8 +955,12 @@ SIMPLE_COMMENT
     : '--' ~[\r\n]* '\r'? '\n'? -> channel(HIDDEN)
     ;
 
+BRACKETED_EMPTY_COMMENT
```

Comment:
Can you try to add this as a case (`| '/**/' -> channel(HIDDEN)`) to the `BRACKETED_COMMENT` rule?